### ISSUE TITLE
[syncfusion_flutter_gauges] callback for onChangeEnd

### ISSUE THREAD

**Issue by scalz** [syncfusion/flutter-widgets#290]
Hello,

At the moment (if I'm not wrong), gauges package provides onChanged callback.
Would it be possible in the future to also have onChangeEnd too ?

Example usecase would be to send only once a http request to my server instead of multiple when moving the cursor on a linear gauge.

Thank you for your work on your packages, they are really nice!

**Comment by LakshmiRadhaKrishnan** [comment]
Hi Scalz, 
 
We have considered your request as a feature and the status can be tracked from the feedback portal link.    
  
**Feedback:** https://www.syncfusion.com/feedback/26450/callbacks-to-identify-the-start-and-end-of-interactions-in-maker-and-widget 
 
This feature will be included in our Volume 3 2021 release, which is expected to be release at the end of September. 
    
If you have any more specification/suggestions to the feature request, you can add it as a comment in the portal.   
 
**Disclaimer:** The date for this feature implementation is tentative and not a commitment on our parts.  
 
Regards, 
Lakshmi R.

**Comment by scalz** [comment]
Awesome, thank you !
